tasks/broad/IlluminaGenotypingArrayTasks.wdl

```diff
@@ -522,6 +522,13 @@ task AutoCall {
 
   String gtc_filename = "~{chip_well_barcode}.gtc"
 
+  # This is the autocall_version, It is normally output by autocall (gencall) itself, except for the case
+  # where autocall fails (likely due to normalization errors)
+  # In this case it no longer emits the version in its output, so we have it here so that it can be output and
+  # stored in the database.
+  # NB - this should be returned from the docker ideally.
+  String autocall_ver = "3.0.0"
```

Comment:
Is there a way to read this from the software in the docker instead of hard-coding it?

Reply:
Unfortunately, it doesn't look like we can.  If autocall succeeds and generates a GTC file, you can query it for the autocallVersion `gtcFile.getAutoCallVersion()` - that's where we get the '3.0.0' for passing runs.  But if autocall fails, it doesn't produce a gtc.  If you query IAAP (the program that runs on the docker and is running autocall) - you get a different version number:
`root@786be8f0813f:/usr/gitc# /usr/gitc/iaap/iaap-cli/iaap-cli -v
1.1.0-1.1.0-sha.80d7e5b3d9c1fdfc2e99b472a90652fd3848bbc7`

Reply:
fair enough